KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -52,6 +52,16 @@ export KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=true
 export KUBEVIRT_DEPLOY_GRAFANA=true
 ```
 
+#### start cluster with ksm enabled
+To enable KSM (Kernel Samepage Merging), please also export the following variables before running `make cluster-up`:
+```bash
+# to tune ksm:
+# KUBEVIRT_KSM_SLEEP_BETWEEN_SCANS_MS - This parameter controls
```

Comment:
This sounds like we *have to* export these. Do sane default values exist that we can use somehow? Maybe we can document those here, if they do exist, or propose something that makes sense in the parameter section below?

Reply:
Done.
Thanks